css/style.css

```diff
@@ -77,3 +93,197 @@ a {
   display: flex;
   justify-content: space-between;
 }
+
+.social__icons li img {
+  width: 19.99px;
+  height: 19.99px;
+}
+
+.action__button {
+  margin-bottom: 20px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+/* work section */
+#work .work-container {
+  display: grid;
+  row-gap: 20px;
+  grid-template-columns: 327px;
+  grid-template-rows: 170px 474px 474px 474px 474px 474px 474px;
+  justify-content: center;
+}
+
+#work .work-container .work-title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  text-transform: capitalize;
+  width: 327px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-image: linear-gradient(#36b37e, #36b37e);
+  background-size: 48px 4px;
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  margin-bottom: 50px;
+}
+
+#work .work-container .work-card {
+  border-radius: 8px;
+  border: 1px solid #d0d9d4;
+}
+
+.work-card .work-img {
+  height: 252px;
+  width: 325px;
+}
+
+.work-card .text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 14px;
+}
+
+.project-title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  color: #3a4a42;
+  text-align: center;
+}
+
+.text .tag {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.text .tag li {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  list-style: none;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+button {
+  background: #36b37f;
+  border-radius: 4px;
+  padding: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border: none;
+}
+
+/* about section */
+#about {
+  background-image: url(photo/illustration\ 1-\ about\ me-mobile.svg);
+  background-repeat: no-repeat;
+  background-position: top right;
+}
+
+#about .about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 333px;
+  margin: 60px 0;
+  padding: 0 25px;
+  background-image: url(photo/illustration-\ about\ me-\ mobile.svg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
```

Comment:
- [ ] Great job with your design 💪 . Kindly make sure the `background-image` in the `About Me` section displays 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205075998-9a49bcac-4afa-4eb6-a1d7-6d6fb85f9989.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205075818-eac71492-797f-47f0-bc86-e5fcd953b967.png)

Reply:
resolved the issue